concepts/docstrings/about.md

```diff
@@ -1,12 +1,16 @@
 # About
 
+A docstring is a piece of text that describes the functionality of the subsequent code.
+Docstrings allow for the automatic generation of documentation.
+They differ from comments in that they're part of the source tree and accessible to the program itself at runtime.[1]
```

Comment:
I think this is confusing in the Julia context. We can also easily look up comments and implementation with `@edit`.

I think we should just say that they differ because they're integrated with the repl and documenter.jl help systems.

Reply:
Only commenting on the last line.

Reply:
```suggestion
They differ from comments in that they are integrated with the REPL and documentation help systems. (More details on those later.)
```